### DISCUSSION TITLE
Tools Function

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#800] [__Dismissed__] [answered]
just like, axe - hoe functions to change/transform blocks, would be cool to have it for shovels, pickaxes, swords, or even shears. Since they added shears feature to cut weeping, twisting and cave vines... would be cool to have these tools

**Comment by BillyGalbreath** [comment, marked as answer]
already in the works.

https://github.com/PurpurMC/Purpur/discussions/449
